### PR TITLE
add language toggle

### PR DESCRIPTION
**Pull request by chrisli-03**
## Type of Pull Request

- [ ] Bug fix
- [x] Enhancement

Related Issue #s or links (if any): https://github.com/developertools-tech/developertools.tech/issues/68

## Description of Changes
add language toggle on nav

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -56,6 +59,68 @@ function Logo({ title }: { title: string }) {
...
+          },
+        }}
+      >
+        {(router.locales || []).map((locale) => (
+          <MenuItem key={locale}>
+            <Link
+              href={router.asPath}
+              locale={locale}
+            >
+              {locale}
```
Comment (dlford) [inline components/Layout.tsx:115]: Can we capitalize these here in the list as well?

**Review by dlford — CHANGES_REQUESTED** [review]
@chrisli-03 great job on this! Just one minor change if you don't mind.

Also, don't worry about the failing test, I'm working on a fix for that.

File: components/Layout.tsx (hunk 1 above)
Comment (chrisli-03) [reply components/Layout.tsx:116]: updated

**Review by dlford — APPROVED** [review]
Awesome, thank you!

**Pull request merged**